sql/civicrm_data/civicrm_option_group/website_type.sqldata.php

```diff
@@ -6,13 +6,6 @@
   ->addValueTable(['name', 'value'], [
     ['Work', 1, 'is_default' => 1],
     ['Main', 2],
```

Comment:
Also I think a label param is missing for these first two, and the label would have ts().

Reply:
How odd. Ok I've added the ts().